src/overlay006/ov6_02240C9C.c

```diff
@@ -753,19 +727,11 @@ static BOOL ov6_02241674 (FieldSystem * fieldSystem, Pokemon * param1, BattlePar
 
 static BOOL ov6_0224174C (FieldSystem * fieldSystem, Pokemon * param1, BattleParams * param2, UnkStruct_ov6_0224222C * param3, const WildEncounters_FieldParams * param4)
 {
-    BOOL v0;
-    {
-        {
-            v0 = ov6_02241DC4(param1, 0xff, param4, param3, ENCOUNTER_TYPE_GRASS, 1, param2);
-
-            if (!v0) {
-                return FALSE;
-            }
-
-            v0 = ov6_02241DC4(param1, 0xff, param4, param3, ENCOUNTER_TYPE_GRASS, 3, param2);
-        }
+    if (!ov6_02241DC4(param1, 0xff, param4, param3, ENCOUNTER_TYPE_GRASS, 1, param2)) {
+        return FALSE;
     }
 
+    BOOL v0 = ov6_02241DC4(param1, 0xff, param4, param3, ENCOUNTER_TYPE_GRASS, 3, param2);
     return v0;
```

Comment:
**question:** Can you not just return the result?

Reply:
No, removing `v0` does not match. I don't really understand why, though. It often seems inconsistent which refactoring match and which don't.

Reply:
How odd. I won't block over this then.